var/spack/repos/builtin/packages/samurai/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Samurai(CMakePackage):
+    """Intervals coupled with algebra of set to handle adaptive
+    mesh refinement and operators on it"""
+
+    homepage = "https://github.com/hpc-maths/samurai"
+    url = "https://github.com/hpc-maths/samurai.git"
+
+    version("master", git=url)
```

Comment:
```suggestion
    version("master", branch="master")
```

Once you set the `git` property in the package class it doesn't need to be reiterated here. However, we should specify the branch to use for the matching version like so.